modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,954 @@
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_nr_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_rnaseq_support_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_gb_rnaseq_nr_'.$self->o('release_number'),
+      -host   => $self->o('genblast_rnaseq_support_db_server'),
+      -port   => $self->o('genblast_rnaseq_support_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'ig_tr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_igtr_'.$self->o('release_number'),
+      -host   => $self->o('ig_tr_db_server'),
+      -port   => $self->o('ig_tr_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'best_targeted_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_bt_'.$self->o('release_number'),
+      -host   => $self->o('genewise_db_server'),
+      -port   => $self->o('genewise_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    long_read_final_db => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_lrfinal_' . $self->o('release_number'),
+      -host   => $self->o('long_read_final_db_server'),
+      -port   => $self->o('long_read_final_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'selected_projection_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_sel_proj_' . $self->o('release_number'),
+      -host   => $self->o('selected_projection_db_server'),
+      -port   => $self->o('selected_projection_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'rnaseq_for_layer_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_rnalayer_nr_' . $self->o('release_number'),
+      -host   => $self->o('rnaseq_for_layer_db_server'),
+      -port   => $self->o('rnaseq_for_layer_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'rnaseq_for_layer_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_rnalayer_nr_'.$self->o('release_number'),
+      -host   => $self->o('rnaseq_for_layer_db_server'),
+      -port   => $self->o('rnaseq_for_layer_db_port'),
```

Comment:
Could this be changed to rnaseq_for_layer_nr_db instead of rnaseq_for_layer_db? This is to allow more flexibility. You would also need to create two entries around the lines 90 similar to the entries it is replacing